src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -281,9 +283,9 @@ private function process_directives_args( string $html, array &$context_stack, a
 					}
 
 					/*
-					 * If this tag will visit its closer tag, it adds it to the tag stack
-					 * so it can process its closing tag and check for unbalanced tags.
-					 */
+					* If this tag will visit its closer tag, it adds it to the tag stack
+					* so it can process its closing tag and check for unbalanced tags.
+					*/
```

Comment:
Should be revert.

Reply:
Thank you! Done in [this commit](https://github.com/WordPress/wordpress-develop/pull/6247/commits/f2dd8b64448cb670665e4b1b8bc417343f7db435).